<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlashCardsMainWindow</class>
 <widget class="QMainWindow" name="FlashCardsMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlashCardsMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>1820</width>
      <height>980</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 135, 135);</string>
    </property>
    <widget class="QWidget" name="previous_card" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>350</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);

 border-radius: 25px;
</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="previous_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>350</width>
        <height>120</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="actual_card" native="true">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>190</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);
border-radius: 25px;</string>
     </property>
     <widget class="QPushButton" name="Submit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 127);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="actual_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>450</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="actual_lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>450</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom: 1px solid;
border-color: rgb(0, 85, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="actual_count">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>490</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="traffic_light" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>540</y>
        <width>451</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 50, 50);
border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="correctAnsware">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>450</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="next_card" native="true">
     <property name="geometry">
      <rect>
       <x>1320</x>
       <y>290</y>
       <width>350</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);
border-radius: 25px;</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="next_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>350</width>
        <height>120</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="previousCardButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>490</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25px;
background-color: rgb(0, 85, 127);</string>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nextCardButton">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>490</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25px;
background-color: rgb(0, 85, 127);</string>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFlashCards">
    <property name="title">
     <string>FlashCards</string>
    </property>
    <addaction name="actionImport_Cards"/>
    <addaction name="actionNew_Card"/>
    <addaction name="actionConnect_to_database"/>
   </widget>
   <addaction name="menuFlashCards"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Cards">
   <property name="text">
    <string>Import Cards</string>
   </property>
  </action>
  <action name="actionNew_Card">
   <property name="text">
    <string>New Card</string>
   </property>
  </action>
  <action name="actionConnect_to_database">
   <property name="text">
    <string>Connect to database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
